src/main/java/com/offdk/play/model/game/MatchPlayer.java

```diff
@@ -7,28 +7,29 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
+import com.offdk.play.model.slack.User;
 import com.offdk.play.rating.RatingType;
 
 import io.vavr.control.Try;
 
 public class MatchPlayer {
-  private final String id;
+  private final User user;
   private final MatchPlayerType type;
   private Optional<Integer> score;
   private Optional<BigDecimal> rating;
 
   // Needed to allow spring-data to serialize data to object
   @PersistenceConstructor
-  MatchPlayer(String id, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
-    this.id = id;
+  MatchPlayer(User user, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
+    this.user = user;
     this.type = type;
     this.score = score;
     this.rating = rating;
   }
 
   @VisibleForTesting
```

Comment:
It is not the fact that it is difficult to store.  It is the fact that match player has mutable attributes that doesn't apply to player